internal/app/init/pkg/system/system_test.go

```diff
@@ -30,3 +30,13 @@ func (suite *SystemServicesSuite) TestStartShutdown() {
 func TestSystemServicesSuite(t *testing.T) {
 	suite.Run(t, new(SystemServicesSuite))
 }
+
+func (suite *SystemServicesSuite) TestStartStop() {
+	system.Services(nil).Start(
```

Comment:
as `system.Services()` returns a singleton instance, there's a bit of a problem with these tests as they depend on the same instance actually. I hope it's fine for these tests, but I'm not sure. iirc there was no way to call `Shutdown()` twise. I was planning to make non-singleton version available for the tests to keep tests isolated, but I haven't got there